### ISSUE TITLE
[Toolbar design][Accessibility] Previous Tab and Next Tab buttons are missing

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23954]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the New Toolbar iOS experiment (New Tab Button and Release)
- Found on Firefox Beta iOS app 134.0

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open any webpage
3. Move VoiceOver focus through the entire URL bar from beginning to end. Observe focus and output.

### Expected behavior / Actual behavior

Per the [Toolbar Redesign Figma](https://www.figma.com/design/8fsCvtnweBkWGZ9NfL907l/__-Toolbar-Redesign?node-id=26560-190307&t=VFHIkB53qYbKUmS9-4), there are buttons next to the Privacy & Security Settings icon button and the refresh icon button for the Previous Tab and Next Tab. These buttons are not currently implemented and won't be until 2025 per Andres Furlan (per conversation in Slack)

+ additional note: looking at the 'touch target areas' in Figma, I believe this should be looked into and that it doesn't look like it meets the [24px by 24px minimum touch target size per WCAG](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)

### Device & build information
* Device: iPhone 15, iOS 18.1.1
* Build version: Firefox Beta 134.0
* First seen version: First time I've seen this issue
### Notes
Attachments: 

https://github.com/user-attachments/assets/937a0e16-9fff-4a4f-9975-e9a82d6d612f

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10968)

**Comment by natastro** [comment]
confirmed with the Andy Furlan that these buttons are not yet implemented for everyone, thus closing this accessibility bug
